Model/Service/Cache/CacheService.php

```diff
@@ -80,14 +81,17 @@ public function __construct(
     /**
      * @param NostoProductInterface $nostoProduct
      * @param StoreInterface $store
+     * @param int|null $lifeTime
      */
-    public function save(NostoProductInterface $nostoProduct, StoreInterface $store)
+    public function save(NostoProductInterface $nostoProduct, StoreInterface $store, $lifeTime)
     {
         try {
             $serializedNostoProduct = $this->productSerializer->toString($nostoProduct);
             $this->productDataCache->save(
                 $serializedNostoProduct,
-                $this->generateCacheKey($nostoProduct->getProductId(), $store->getId())
+                $this->generateCacheKey($nostoProduct->getProductId(), $store->getId()),
+                [ProductData::CACHE_TAG],
```

Comment:
No need to pass the cache_tag as it's passed by the method itself in https://github.com/magento/magento2/blob/2.4-develop/lib/internal/Magento/Framework/Cache/Frontend/Decorator/TagScope.php#L51

Reply:
Removed